ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/stun.yml

```
@@ -0,0 +1,10 @@
+### stun
+---
+stun:
+  server:
+    local_interface: ethernet1
+  client:
+    - server_profile: server1
+      ip_address: 1.2.3.4
+    - server_profile: server2
+      ip_address: 2.3.4.5
```

This is the correct pattern for the schema ;)

```suggestion
    server_profiles:
      - name: server1
         ip_address: 1.2.3.4
      - name: server2
         ip_address: 2.3.4.5
```